### PR TITLE
Remove a workaround for badly-printed dot products

### PR DESCRIPTION
**Pull request by eric-wieser**
There is no need to post-process latex to solve this, it's now solved for us in `galgebra.atoms.DotProductSymbol`.

### CODE REVIEW

File: examples/ipython/LaTeX.ipynb [hunk 1]
```diff
@@ -73,9 +73,9 @@
...
+       "\\begin{equation*} (-v (x_1\\W x_2)/2).exp()== \\cos{\\left (\\frac{\\left|{v}\\right|}{2} \\right )}  - \\frac{v \\sin{\\left (\\frac{\\left|{v}\\right|}{2} \\right )}}{2 \\left|{v}\\right|} \\boldsymbol{e}_{1}\\wedge \\boldsymbol{e}_{2} - \\frac{v \\sin{\\left (\\frac{\\left|{v}\\right|}{2} \\right )}}{2 \\left|{v}\\right|} \\boldsymbol{e}_{1}\\wedge \\boldsymbol{e}_{4} + \\frac{v \\sin{\\left (\\frac{\\left|{v}\\right|}{2} \\right )}}{2 \\left|{v}\\right|} \\boldsymbol{e}_{2}\\wedge \\boldsymbol{e}_{3} - \\frac{v \\sin{\\left (\\frac{\\left|{v}\\right|}{2} \\right )}}{2 \\left|{v}\\right|} \\boldsymbol{e}_{3}\\wedge \\boldsymbol{e}_{4} \\end{equation*}\n",
```
Comment (eric-wieser) [inline examples/ipython/LaTeX.ipynb:76]: Changed from `) \cdot exp` to `).exp`

File: examples/LaTeX/Dop.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 f = o3d.mv('f', 'scalar', f=True)
 lap = o3d.grad*o3d.grad
 print(r'\nabla =', o3d.grad)
-print(r'%\nabla^{2} = \nabla . \nabla =', lap)
+print(r'%\nabla^{2} = \nabla \cdot \nabla =', lap)
```
Comment (eric-wieser) [inline examples/LaTeX/Dop.py:13]: The percent is supposed to mean "do not postprocess this", as far as I can tell.

At any rate, this line already contains latex, so there's no point translating only bits of it.

File: galgebra/printer.py [hunk 3]
```diff
@@ -819,11 +819,6 @@ def tex(paper=(14, 11), debug=False, prog=False, pt='10pt'):
                     latex_line = '\\begin{equation*} ' + latex_line + ' \\end{equation*}\n'
 
         else:
-            latex_line = latex_line.replace(r'\left.', r'@@')  # Disabiguate '.' in '\left.'
-            latex_line = latex_line.replace(r'\right.', r'##')  # Disabiguate '.' in '\right.'
-            latex_line = latex_line.replace('.', r' \cdot ')  # For components of metric tensor
-            latex_line = latex_line.replace(r'@@', r'\left.')  # Restore '\left.'
-            latex_line = latex_line.replace(r'##', r'\right.')  # Restore '\right.'
```
Comment (utensil) [inline galgebra/printer.py:826]: Although I do agree to remove these erroneous bits but it's a breaking change so I marked this PR as "need changelog"

Comment (eric-wieser) [reply galgebra/printer.py:826]: Agreed :)

**Pull request merged**